_docs/tasks/traffic-management/ingress.md

```diff
@@ -269,6 +269,204 @@ rules.
    curl -I -k https://$INGRESS_HOST/status/200
    ```
 
+1. Configuring RBAC for ingress key/cert.
+
+   There are service accounts which can access this ingress key/cert, and this leads to risks of 
+   leaking key/cert. We can set up Role-Based Access Control ("RBAC") to protect it.
+   [istio/install/kubernetes/istio.yaml](https://github.com/istio/istio/blob/master/install/kubernetes/istio.yaml)
+   defines ClusterRoles and ClusterRoleBindings which allow service accounts in namespace istio-system
+   to access all secret resources. We need to update or replace these RBAC set up to only allow
+   istio-ingress-service-account to access ingress key/cert.
+
+   We can use kubectl to list all secrets in namespace istio-system that we need to protect using RBAC.
+   
+   ```bash
+   kubectl get secrets -n istio-system
+   ```
+   This produces the following output:
+   
+   ```bash
+   NAME                                        TYPE                                  DATA      AGE
+   istio-ca-secret                             istio.io/ca-root                      2         7d
+   istio-ingress-certs                         kubernetes.io/tls                     2         7d
+   istio-ingress-service-account-token-kwr85   kubernetes.io/service-account-token   3         7d
+   istio.istio-ingress-service-account         istio.io/key-and-cert                 3         7d
+   ......
+   ```
+   
+1. Update RBAC set up for istio-pilot-service-account and istio-mixer-istio-service-account.
+   
+   Delete existing ClusterRoleBindings and ClusterRole.  
+
+   ```bash
+   kubectl delete ClusterRoleBinding istio-pilot-admin-role-binding-istio-system
+   kubectl delete ClusterRoleBinding istio-mixer-admin-role-binding-istio-system
+   kubectl delete ClusterRole istio-mixer-istio-system
+   ```
+   
+   Create istio-pilot-mixer-istio-system.yaml, which allows istio-pilot-service-account and 
+   istio-mixer-service-account to read all kubernetes.io/service-account-token, 
```

Comment:
I don't think anyone needs to read `kubernetes.io/service-account-token`, no?

Reply:
Agree. That line is removed.